### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - Evan Rothen

### DISCUSSION THREAD

**Discussion by EVAN-ROTHEN** [heig-vd-dai-course/heig-vd-dai-course#383] [Show and tell]
https://github.com/EVAN-ROTHEN/04-java-intellij-idea-and-maven

**Comment by ludelafo** [comment]
Thank you!

It seems the `pom.xml` file is missing some properties in the `properties` key. There should be some default values you need to keep when you first create the project.

You can always find the solution in the solution repository if you ever need to compare your work with the official one here: https://github.com/heig-vd-dai-course/heig-vd-dai-course-solutions.

Do not hesitate to ask questions if something is not clear, either in this discussion or in the Q&A category of the GitHub Discussions here: https://github.com/orgs/heig-vd-dai-course/discussions/categories/q-a.
